packages/blockchain-wallet-v4-frontend/src/data/modules/addressesBch/sagas.js

```diff
@@ -24,7 +28,54 @@ export default ({ coreSagas }) => {
     }
   }
 
+  const selectChangeAddresses = (accountIndex, start, end, state) => {
+    let addrs = []
+    for (var i = start + GAP_LIMIT; i < end; i += GAP_LIMIT) {
+      // get address at GAP_LIMIT points
+      const addr = selectors.core.common.bch.getAddress(
+        networks.btc,
+        `${accountIndex}/${1}/${i}`,
+        state
+      )
+      addrs.push(addr)
+    }
+    return addrs
+  }
+
+  // if change was sent to users btc change index instead of bch
+  const showBchChangeAddrs = function*(action) {
+    const { index, xpub } = action.payload
+    const state = yield select()
+    const btcChangeIndex = selectors.core.data.bitcoin
+      .getChangeIndex(xpub, state)
+      .getOrElse(0)
+    const bchChangeIndex = selectors.core.data.bch
+      .getChangeIndex(xpub, state)
+      .getOrElse(0)
+
+    // check if bch change index got behind btc by gap limit
+    const addrs = selectChangeAddresses(
+      index,
+      bchChangeIndex,
+      btcChangeIndex,
+      state
+    )
+
+    // if there are addrs to display show them else carry on
+    /* eslint-disable */
+    if (addrs.length) {
+      window.alert(
```

Comment:
nice solution although these might be blocked by browser, console.log is safer imo

Reply:
We're a bit worried users won't be able to easily open the console. I'll check in with support after the fix is deployed, if anyone has issues we'll switch to console log.